sockeye/rnn_attention.py

```diff
@@ -59,6 +63,26 @@ def __init__(self,
         self.layer_normalization = layer_normalization
         self.config_coverage = config_coverage
         self.num_heads = num_heads
+        self.dtype = dtype
+
+
+def _instantiate(cls, params):
+    """
+    Helper to instantiate Attention classes from parameters. Warns in log if parameter is not supported
```

Comment:
Not really specific to Attention is it? Would it make sense to move this to utils.py and generalize the doc string (and the additional args you pass, e.g. max_seq_len)?

Reply:
sure, fine